src/ol/ol.css

```diff
@@ -111,6 +111,10 @@
   all: unset;
 }
 
+.ol-viewport * {
```

Comment:
Why not just
```suggestion
.ol-viewport {
```
?

Reply:
The setting is not inherited and in #13495 the main problem is large controls within the viewport.  I'm not entirely sure what is causing the problem in #14893 but I suspect transient empty elements as the layer visibility changes.

Reply:
I think the `*` is too invasive for a library setting to make.

Reply:
`.ol-viewport` alone will fix #14893 and the full screen mobile example in #13495 as one finger is going to be on the viewport.  If an application has controls such as layer switcher or geocoder then either the control or application will need to suppress those from scaling the page.

Reply:
I'd say this is the responsibility of the application. Because as a user, I might expect to be able to zoom the output of a layer switcher or geocoder.

In the first place
html```
<meta name="viewport" content="initial-scale=1.0, user-scalable=no, width=device-width">
```
in the document's head should prevent user zooming on mobile devices.